charts/cmc-citizen-frontend/values.aat.template.yaml

```diff
@@ -0,0 +1,10 @@
+nodejs:
+  environment:
+    IDAM_API_URL: https://idam-api.aat.platform.hmcts.net
+    IDAM_AUTHENTICATION_WEB_URL: https://idam-web-public.aat.platform.hmcts.net
+    IDAM_S2S_AUTH: http://rpe-service-auth-provider-aat.service.core-compute-aat.internal
```

Comment:
You need them in .preview.template.yaml as well We have moved them to values.yaml to be sensible defaults.?

Reply:
Same with other URLs as well if they are applicable?